utils/api.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
+
+const newsApi = axios.create({
+    baseURL: 'https://hjeffs-nc-news.onrender.com/api',
+})
+
+export const getArticles = (item_id) => {
+    return newsApi
+    .get('/articles', { params: { article_id: item_id } })
+    .then((response) => {
+        // console.log('getArticles')
+        // console.log(response.data, '<< .data')
+        // console.log(response.data.articles, '<<<.articles')
+        return response.data.articles
+    })
+}
+
+export const getArticleById = (item_id) => {
+    // console.log(item_id);
+    return newsApi
+        .get(`/articles/${item_id}`)
+        .then((response) => {
+            // console.log(response.data.article[0])
```

Comment:
Whenever you are creating a PR, you are saying "this piece of work is done" - so you definitely want to get rid of console.logs and unnecessary comments. Here, both of these things combine, so it should be going 😎